README.md

```diff
@@ -564,6 +683,18 @@ const { result, error } = await deepgram.manage.getTokenDetails();
 
 [See our API reference for more info](https://developers.deepgram.com/reference/authentication)
 
+### Grant Token
+
+Creates a temporary token with a 30-second TTL.
+
+```js
+const { result, error } = await deepgram.auth.grantToken();
+```
+
+[See our API reference for more info](https://developers.deepgram.com/reference/token-based-auth-api/grant-token).
+
+[See our Example for info](). @TODO See if we need an example
```

Comment:
@naomi-lgbt  @lukeocodes  Do you want me to add an example here? We currently don't have one for this case.